lib/locreator.js

```diff
@@ -70,10 +70,10 @@ Locreator.prototype.addStarMethods = function addStarMethods(locatorId, locatorJ
     return drivex.present(locator(), parentWebElement);
   };
   locatorObject[locatorId + 'Wait'] = function (timeout, msg) {
-    return drivex.waitForElementPromise(locator(), timeout || 5000, msg);
+    return drivex.waitForElementPromise(locator(), timeout || 5000, msg || 'nemo-view: Wait failed for locator [' + locatorId + ']');
   };
   locatorObject[locatorId + 'WaitVisible'] = function (timeout, msg) {
-    return drivex.waitForElementVisiblePromise(locator(), timeout || 5000, msg);
+    return drivex.waitForElementVisiblePromise(locator(), timeout || 5000, msg || 'nemo-view: WaitVisible failed for locator [' + locatorId + ']');
```

Comment:
You can remove "nemo-view: " from the message. The logger will prepend the module name.